DOCUMENTATION.md

```diff
@@ -96,8 +96,20 @@ Sends a downlink to the device.
 client.send(dev_id, payload, [port], [confirmation], [schedule])
 ```
 - `dev_id`: **string**  the ID of the device you wish to send the message to.
-- `payload`: the payload of the message to be published to the broker. It can be an hexadecimal **string**, a base64 **string** like `AQ==` (this will send the raw payload `01` to your device) or an object with several fields following the **JSON** standard. In case it's a **JSON** object with fields, please make sure the **encoder** function (Payload Formats section) of the application is set to make sense of the informations transmitted in each field.
+- `payload`: the payload of the message to be published to the broker. It can be an hexadecimal **string**, a base64 **string** like `AQ==` (this will send the raw payload `01` to your device) or an object in **JSON**. Here is an example of a **JSON** argument that could be passed to the method:
```

Comment:
JSON is not a type in Python - please mention the first-class type (`dict`/`string`) expected, then mention the JSON nature of that variable